### PR TITLE
Directional NDBC 

### LINKED ISSUE #191: Feature Request: Directional Spectra from NDBC buoys
A comment raised in OREC workshop was about getting directional spectral data from NDBC and it sounded like there was some code available. I would be interested in learning how to do that and helping to create a function.

Thanks!

### PR DESCRIPTION
**Pull request by cmichelenstrofer**
resolves #191

### CODE REVIEW

**Comment by cmichelenstrofer** [comment]
@ssolson @akeeste This should be ready for you to review. Thanks! :)

**Review by akeeste — APPROVED** [review]
@cmichelenstrofer Thanks for putting this together! I only have a couple minor comments on docstrings and one test, but otherwise this looks good!

**Comment by ssolson** [comment]
@cmichelenstrofer when you get a chance could you address Adam's points above and merge the latest Development branch with this PR?

File: examples/environmental_contours_example.ipynb [hunk 1]
```diff
@@ -20,7 +20,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
```
[the remark this answers is not in the document]
Comment (cmichelenstrofer) [reply examples/environmental_contours_example.ipynb:24]: yeah, most examples are already run, but this one wasn't. I like being able to go to the repo and see the plots without having to run locally (which is a pin with `requests` and proxies...)

**Comment by cmichelenstrofer** [comment]
@akeeste @ssolson This should be ready. @ssolson I like what you did to the tests! Much more manageable in smaller files like that.

**Review by akeeste — APPROVED** [review]
@cmichelenstrofer thanks for the update! Changes approved on my end.

**Pull request merged**